src/CLIPreChecker/checkForDeprecate.ts

```diff
@@ -0,0 +1,19 @@
+import { NpmClient } from '../clients/npmClient'
+import { CLIPrecheckerStore, ICLIPrecheckerStore } from './CLIPrecheckerStore'
+
+export const checkForDeprecate = async (store: ICLIPrecheckerStore, pkgName: string, pkgVersion: string) => {
+  try {
+    const { deprecated } = await NpmClient.getPackageMetadata(pkgName, pkgVersion)
+    store.setDeprecated(deprecated != null)
+    store.setLastDeprecationCheck(Date.now())
+    process.exit()
+  } catch (err) {
+    console.log(err)
```

Comment:
Why not using `log.error` from `logger`?

Reply:
Ooops, it's better to use the logger. Will change it